queue/src/fiskaltrust.Middleware.Localization.QueueIT/v2/Lifecycle/FinishSCUSwitch0x4012.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using fiskaltrust.ifPOS.v1;
+using fiskaltrust.Middleware.Localization.QueueIT.Constants;
+using fiskaltrust.storage.V0;
+
+namespace fiskaltrust.Middleware.Localization.QueueIT.v2.Lifecycle
+{
+    public class FinishSCUSwitch0x4012 : IReceiptTypeProcessor
+    {
+        public ITReceiptCases ReceiptCase => ITReceiptCases.FinishSCUSwitch0x4012;
+
+        public async Task<(ReceiptResponse receiptResponse, List<ftActionJournal> actionJournals)> ExecuteAsync(ftQueue queue, ftQueueIT queueIt, ReceiptRequest request, ReceiptResponse receiptResponse, ftQueueItem queueItem) => await Task.FromResult((receiptResponse, new List<ftActionJournal>())).ConfigureAwait(false);
```

Comment:
I feel like we should return state `EEEE_EEEE` here.

This receipt is normally part of a real switching process and someone might expect this to do anything and do something illegal? (idk how the situation in italy is regarding changing printers).

Or do we expect them to go through he switch process anyways? I feel like in this case we need to make this do something that ensures they really use it ^^